test/controllers/v1/events_controller_test.rb

```diff
@@ -60,4 +60,19 @@ def teardown
     end
     assert_response :not_found
   end
+
+  test "alert when trying to create an invalid event" do
+    missing_name = { date: Time.zone.now, owner: @logged_user }
+    assert_no_difference 'Event.count' do
+      get :create, missing_name.merge!(@auth_params)
+    end
+    assert_response :forbidden
+    assert_equal response.body, %Q({"success":false,"message":["Name can't be blank"]})
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.